gammapy/estimators/parameter.py

```diff
@@ -262,6 +274,27 @@ def estimate_ul(self, datasets, parameter):
         )
         return {f"{parameter.name}_ul": res["errp"] + parameter.value}
 
+    def estimate_sensitivity(self, datasets, parameter):
+        """Estimate norm sensitivity for the flux point.
+
+        Parameters
+        ----------
+        datasets : `~gammapy.datasets.Datasets`
+            Datasets.
+
+        Returns
+        -------
+        result : dict
+            Dictionary with an array with one entry per dataset with the sum of the
+            masked npred.
```

Comment:
can you describe the content of the dictionary? One needs to be now more explicit and clear for the dict returned values, following users feedbacks